data/src/main/java/com/moonpig/data/ProductDetailStubRepository.kt

```diff
@@ -20,6 +26,12 @@ class ProductDetailStubRepository : ProductDetailRepository {
                     )
             )
 
+    private fun respondWithLoadError() =
+            Observable.just(
+                    RepositoryState.GetProductDetail.InFlight,
```

Comment:
Could we put a timer on this to simulate network latency, so we see InFlight for say 2 secs, then error?